packages/vm/src/runCall.ts

```diff
@@ -74,14 +78,23 @@ export interface RunCallOpts {
 /**
  * @ignore
  */
-export default function runCall(this: VM, opts: RunCallOpts): Promise<EVMResult> {
+export default async function runCall(this: VM, opts: RunCallOpts): Promise<EVMResult> {
   const block = opts.block ?? Block.fromBlockData({}, { common: this._common })
 
   const txContext = new TxContext(
     opts.gasPrice ?? BigInt(0),
     opts.origin ?? opts.caller ?? Address.zero()
   )
 
+  if (opts.skipBalance) {
```

Comment:
Can you add a comment explaining, like...

```suggestion
  // if skipBalance, ensure the caller has enough balance for `value` transfer
  if (opts.skipBalance) {
```